docs/content/preview/explore/query-1-performance/pg-stat-progress-create-index.md

```diff
@@ -0,0 +1,100 @@
+
+- Number of rows processed during an index backfill.
+- The current phase of the command with `initializing` or `backfilling` as the possible phases.
+- Index progress report for all the different configurations of an index or index build such as non-concurrent index builds, GIN indexes, partial indexes, and include indexes.
+
+The following table describes the view columns:
+
+| Column | Type | Description |
+| :----- | :--- | :---------- |
+| pid | integer | Process ID of backend that is running the CREATE INDEX. |
+| datid | OID | Object ID of the database to which this backend is connected. |
+| datname | name | Name of the database to which this backend is connected. |
+| relid | OID | Object ID of the indexed relation.|
+| index_relid | OID | Object ID of the index. |
+| command | text | The command that is running CREATE INDEX CONCURRENTLY, or CREATE INDEX NONCONCURRENTLY. |
+| phase | text | The current phase of the command. The possible phases are _initializing_, or _backfilling_. |
+| tuples_total | bigint | Number of indexed table tuples already processed. |
+| tuples_done | bigint | Estimate of total number of tuples (in the indexed table). This value is retrieved from `pg_class.reltuples`. |
+| partitions_total | bigint | If the ongoing CREATE INDEX is for a partitioned table, this refers to the total number of partitions in the table. Set to 0 otherwise. |
+| partitions_done | bigint | If the ongoing CREATE INDEX is for a partitioned table, this refers to the number of partitions the index has been created for. Set to 0 otherwise. |
+
+Columns such as `lockers_total`, `lockers_done`, `current_locker_pid`, `blocks_total`, and `blocks_done` are not applicable to YugabyteDB and always have null values.
+
+## YugabyteDB-specific changes
+
+The `pg_stat_progress_create_index` view includes the following YugabyteDB-specific changes:
+
+- In YugabyteDB, the `pg_stat_progress_create_index` view is a local view; it only has entries for CREATE INDEX commands issued by local YSQL clients.
+
+- In PostgreSQL, `tuples_done` and `tuples_total` refer to the tuples of the _index_. However, in YugabyteDB, these fields refer to the tuples of the _indexed table_. This applies only to partial indexes, where the reported progress is less than the actual progress. `tuples_total` is an estimate that is retrieved from `pg_class.reltuples`.
+
+## Example
+
+The following example demonstrates the possible phases (initializing, backfilling) for the CREATE INDEX operation using the `pg_stat_progress_create_index` view.
+
+{{< note title="Setup" >}}
+
+Local single-node cluster. See [Set up YugabyteDB universe](../../../explore/#set-up-yugabytedb-universe).
+
+{{< /note >}}
+
+1. From your local YugabyteDB installation directory, connect to the [YSQL](../../../admin/ysqlsh/) shell, and create an index on an existing table as follows:
+
+    ```sql
+    CREATE TABLE employees (id int, name text, department text);
```

Comment:
My bad - the example in the doc wasn't correct.
This should be - `CREATE TABLE customers (id int, customer_name text);`